common/app/common/Edition.scala

```diff
@@ -165,9 +165,3 @@ object InternationalEdition {
     }
   }
 }
-
-object InternationalEditionVariant {
-
-  def apply(request: RequestHeader): Option[String] = request.headers.get("X-GU-International")
```

Comment:
This test has been revoked long ago https://github.com/guardian/fastly-edge-cache/commit/a4a9244 :D